### PR TITLE
Detail how minutes and meetings will follow our rules of order.

### PR DESCRIPTION
**Pull request by erinishimoticha**
Detail how minutes and meetings will follow our rules of order.

### CODE REVIEW

File: Natives_In_Tech_Bylaws.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ An Advisory Council may be created whose members shall be elected by the members
 
 **Section 13. Parliamentary Procedure**
 
-Any question concerning parliamentary procedure at meetings shall be determined by the President by reference to Robert's Rules of Order.
+Any question concerning parliamentary procedure shall be determined by the President by reference the organization's [rules of order](Git_Rules_of_Order.md).
```
Comment (arecvlohe) [inline Natives_In_Tech_Bylaws.md]: *by referencing

**Review by arecvlohe — DISMISSED** [review]

File: Natives_In_Tech_Bylaws.md (hunk 1 above)
Comment (erinishimoticha) [reply Natives_In_Tech_Bylaws.md]: Ah I deleted `to`.  Fixed.

**Review by csutherl — DISMISSED** [review]
LGTM

**Review by microfauna — DISMISSED** [review]

File: Natives_In_Tech_Bylaws.md (hunk 1 above)
Comment (arecvlohe) [reply Natives_In_Tech_Bylaws.md]: @erinishimoticha Did you push up your changes?

**Comment by arecvlohe** [comment]
@erinishimoticha It looks like it's ready, based on approvals, but I think you might not have pushed up a change?

File: Natives_In_Tech_Bylaws.md (hunk 1 above)
Comment (erinishimoticha) [reply Natives_In_Tech_Bylaws.md]: Whoops, pushed now.

**Review by arecvlohe — APPROVED** [review]

**Review by microfauna — APPROVED** [review]

**Review by csutherl — APPROVED** [review]

**Pull request merged**